examples/clusterctl-templates/clusterctl-cluster.yaml

```diff
@@ -1,47 +1,56 @@
-apiVersion: cluster.x-k8s.io/v1alpha3
+apiVersion: cluster.x-k8s.io/v1alpha4
 kind: Cluster
 metadata:
   name: ${CLUSTER_NAME}
   namespace: ${NAMESPACE}
 spec:
   clusterNetwork:
     services:
-      cidrBlocks: [${SERVICE_CIDR}]
+      cidrBlocks:
+      - 10.96.0.0/12
     pods:
-      cidrBlocks: [${POD_CIDR}]
+      cidrBlocks:
+      - 192.168.0.0/18
   infrastructureRef:
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
     kind: Metal3Cluster
     name: ${CLUSTER_NAME}
+    namespace: ${NAMESPACE}
   controlPlaneRef:
     kind: KubeadmControlPlane
-    apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+    apiVersion: controlplane.cluster.x-k8s.io/v1alpha4
     name: ${CLUSTER_NAME}
+    namespace: ${NAMESPACE}
 ---
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
 kind: Metal3Cluster
 metadata:
   name: ${CLUSTER_NAME}
   namespace: ${NAMESPACE}
 spec:
   controlPlaneEndpoint:
-    host: ${API_ENDPOINT_HOST}
-    port: ${API_ENDPOINT_PORT}
+    host: 192.168.111.249
+    port: 6443
```

Comment:
is it possible to keep these parametrised ?

Reply:
Sure, done.